docs/reference/ml/df-analytics/apis/put-dfanalytics.asciidoc

```diff
@@ -27,57 +27,46 @@ built-in roles and privileges:
 * `kibana_admin` (UI only)
 
 
-* source index: `read`, `view_index_metadata`
+* source indices: `read`, `view_index_metadata`
 * destination index: `read`, `create_index`, `manage` and `index`
 * cluster: `monitor` (UI only)
   
 For more information, see <<security-privileges>> and <<built-in-roles>>.
 
-+
---
 NOTE: It is possible that secondary authorization headers are supplied in the
 request. If this is the case, the secondary authorization headers are used
 instead of the primary headers.
---
 
 [[ml-put-dfanalytics-desc]]
 ==== {api-description-title}
 
 This API creates a {dfanalytics-job} that performs an analysis on the source 
-index and stores the outcome in a destination index.
-
-The destination index will be automatically created if it does not exist. The 
-`index.number_of_shards` and `index.number_of_replicas` settings of the source 
-index will be copied over the destination index. When the source index matches 
-multiple indices, these settings will be set to the maximum values found in the 
-source indices.
-
-The mappings of the source indices are also attempted to be copied over
-to the destination index, however, if the mappings of any of the fields don't 
-match among the source indices, the attempt will fail with an error message.
-
-If the destination index already exists, then it will be use as is. This makes 
-it possible to set up the destination index in advance with custom settings 
-and mappings.
-
-[discrete]
-[[ml-hyperparam-optimization]]
-===== Hyperparameter optimization
-
-If you don't supply {regression} or {classification} parameters, _hyperparameter 
-optimization_ occurs, which sets a value for the undefined parameters. The
-starting point is calculated for data dependent parameters by examining the loss
+indices and stores the outcome in a destination index.
+
+If the destination index does not exist, it is created automatically when you
+start the job. See <<start-dfanalytics>>.
+
+If you supply only a subset of the {regression} or {classification} parameters,
+_hyperparameter optimization_ occurs. It determines a value for each of the
+undefined parameters. A fixed number of rounds is used for optimization which
+depends on the number of parameters being optimized.
+// TBD: Does this "fixed number of rounds" sentence mean that the more undefined
```

Comment:
@tveasey Could you help us clarify this please?

Reply:
I will pursue this in https://github.com/elastic/stack-docs/pull/990 instead